plugins/graphql-kotlin-plugin-core/src/main/kotlin/com/expediagroup/graphql/plugin/generator/exceptions/InvalidSelectionSetException.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql.plugin.generator.exceptions
+
+/**
+ * Exception thrown when specified query file contains invalid selection set.
+ */
+internal class InvalidSelectionSetException(message: String) : RuntimeException(message)
```

Comment:
This would allow us to have a standard message. It requires an update of all the usages though

```suggestion
internal class InvalidSelectionSetException(typeDefinitionName : String, typeName : String) : RuntimeException("Invalid selection set for $typeName - cannot select empty $typeDefinitionName")
```